src/main/java/IllegalTaskException.java

```diff
@@ -0,0 +1,5 @@
+/**
+ * Throws an exception when starting word that user input is not "todo", "deadline", or "event".
```

Comment:
Good job using JavaDocs ahead of time! However, you are suggested to use a consistent JavaDocs format for similar methods and classes.